packages/bpk-docs/src/layouts/NeoSideNavLayout/SideNavLayout.scss

```diff
@@ -20,6 +20,7 @@
 
 .bpkdocs-side-nav-layout {
   display: flex;
+  flex: 1 0 auto;
```

Comment:
causes the side-nav layout to fill the space available vertically - see Matt's screenshot